client/src/components/Payment/StripeForm.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo } from 'react';
+import { Button, Box, Grid, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import { CardElement, useStripe, useElements } from '@stripe/react-stripe-js';
+import useStyles from './useStyles';
+interface Props {
+  onClose: () => void;
+}
+const StripeForm = ({ onClose }: Props): JSX.Element => {
+  const classes = useStyles();
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const options = useMemo(
+    () => ({
+      style: {
+        base: {
+          fontSize: '1.2rem',
+          color: '#424770',
+          letterSpacing: '0.025em',
+          fontFamily: 'Source Code Pro, monospace',
+          '::placeholder': {
+            color: '#aab7c4',
+          },
+        },
+        invalid: {
+          color: '#9e2146',
+        },
+      },
+    }),
+    [],
+  );
+
+  const handleSubmit = async (event: { preventDefault: () => void }) => {
+    event.preventDefault();
+    if (elements == null) {
+      return;
+    }
+    if (!stripe || !elements) {
+      return;
+    }
+    const cardElement = elements.getElement(CardElement);
+
+    if (!cardElement) return;
```

Comment:
You could check the Truthiness of all the variables in one line and return if any of them is Falsy. For example;
`if (elements && stripe && cardElement) {
   // perform other tasks
  } else { return; }`

Reply:
Should I include the cardElement in if condition? Because it cannot be checked before we get it?